ateam_ssl_simulation_radio_bridge/src/ssl_simulation_radio_bridge_node.cpp

```diff
@@ -67,14 +76,28 @@ class SSLSimulationRadioBridgeNode : public rclcpp::Node
 
     std::string protobuf_msg;
     if (robots_control.SerializeToString(&protobuf_msg)) {
-      udp_sender_.send(protobuf_msg.data(), protobuf_msg.size());
+      udp_.send(protobuf_msg.data(), protobuf_msg.size());
+    }
+  }
+
+  void feedback_callback(const char * buffer, size_t bytes_received)
+  {
+    RobotControlResponse feedback_proto;
+    if (!feedback_proto.ParseFromArray(buffer, bytes_received - 1)) {
```

Comment:
Same issue as the vision filter. It really does not like reading the entire packet. Someone else should probably try this since it may be a WSL issue